src/index.js

```diff
@@ -1,15 +1,32 @@
 var SentryCli = require('@sentry/cli');
 
+var SOURCEMAPS_OPTIONS = [
+  'release',
+  'configFile',
+  'include',
+  'ignore',
+  'ignoreFile',
+  'noSourceMapReference',
+  'stripPrefix',
+  'stripCommonPrefix',
+  'validate',
+  'urlPrefix',
+  'ext'
+];
+
 function SentryCliPlugin(options) {
   options = options || {};
-  this.release = options.release;
-  this.include =
-    options.include &&
-    (Array.isArray(options.include) ? options.include : [options.include]);
-  this.configFile = options.configFile;
-  this.ignoreFile = options.ignoreFile;
-  this.ignore =
-    options.ignore && (Array.isArray(options.ignore) ? options.ignore : [options.ignore]);
+
+  SOURCEMAPS_OPTIONS.forEach(sourceMapOption => {
```

Comment:
The logic here should be reversed. We should iterate through passed options, not all available ones.
You also always overrides previous `if` check and perform unnecessary assignments in ternary operator.

Much simpler version:

```js
const SOURCEMAPS_OPTIONS = [ __list_them_here__ ];
const MULTI_OPTIONS = ['ignore', 'include'];

function SentryCliPlugin(options = {}) {
  Object.keys(options).forEach((option) => {
    if (!SOURCEMAPS_OPTIONS.includes(option)) return;

    let optionValue = options[option];

    if (MULTI_OPTIONS.includes(option)) {
      this[option] = Array.isArray(optionValue) ? optionValue : [optionValue];
    } else {
      this[option] = optionValue;
    }
  });
}
```

As we use ES6, we can also move default assignment to argument `function SentryCliPlugin(options = {}) { ... }`.

Reply:
If statement can also be flipped and we can make it one-liner

```js
this[option] = (MULTI_OPTIONS.includes(option) && !Array.isArray(optionValue))
  ? [optionValue]
  : optionValue;
```

but I don't think it's any more readable really.